src/components/tooltip/Tooltip.css

```diff
@@ -89,3 +89,7 @@
   color: var(--grey-50);
   white-space: nowrap;
 }
+
+.tooltip .tooltipMarker {
+  max-width: 600px;
+}
```

Comment:
I'd like to push back on this change being in this PR. I disagree with this UI change since it's useful to have really long tooltips for markers with long single line pieces of data. This can result in data loss for a user not being able to see all of the information. I'm not sure we handle truncating things gracefully, and would prefer to make sure we are doing the right thing here. I'm open to discussing the change, but I'd prefer not to clutter up this PR with the discussion. If you feel this is important I'd prefer to take this conversation into a new issue or PR.